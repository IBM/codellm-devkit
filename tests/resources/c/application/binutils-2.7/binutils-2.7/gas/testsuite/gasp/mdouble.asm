
	.MACRO	HI
	A
	\! this is hidden
	B
	! this is not
	C
	.ENDM
	Hello 
	HI
	Emily
	

	H'0f
	200+H'0F

XX	.ASSIGNA	Q'100
! Definition:
	.MACRO	GET X=100,Y,Z
	MOV	#\X+H'0F,@B
	\Y
\Z	JMP	@MAIN
L\@	ADD 	#1,@HL
	MOV	#0,@C	\! Clear C
	ADD	#2,@C
	ADD	#\&XX, @C
	.ENDM

	NOP

!Call:	
	GET	200,"ADD #1,@B", ENTRY
	.END

	; Definition:


	NOP

	;Call:	
	MOV	#200+0F,@B
	ADD #1,@B
ENTRY:	JMP	@MAIN
L00000:	ADD 	#1,@HL
	MOV	#0,@C	
	ADD	#2,@C
	ADD	#0, @C
