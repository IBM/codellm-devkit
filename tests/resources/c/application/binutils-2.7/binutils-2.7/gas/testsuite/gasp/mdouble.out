!

!	.MACRO	HI
!	A
!	\! this is hidden
!	B
!	! this is not
!	C
!	.ENDM
!	Hello 
	Hello 
!	HI
!	A
	A
!	
!	B
	B
!	! this is not
	! this is not
!	C
	C
!	Emily
	Emily
!	
!

!	H'0f
	15
!	200+H'0F
	200+15
!

!XX	.ASSIGNA	Q'100
!! Definition:
	! Definition:
!	.MACRO	GET X=100,Y,Z
!	MOV	#\X+H'0F,@B
!	\Y
!\Z	JMP	@MAIN
!L\@	ADD 	#1,@HL
!	MOV	#0,@C	\! Clear C
!	ADD	#2,@C
!	ADD	#\&XX, @C
!	.ENDM
!

!	NOP
	NOP
!

!!Call:	
	!Call:	
!	GET	200,"ADD #1,@B", ENTRY
!	MOV	#200+H'0F,@B
	MOV	#200+15,@B
!	ADD #1,@B
	ADD #1,@B
!ENTRY	JMP	@MAIN
ENTRY:	JMP	@MAIN
!L00001	ADD 	#1,@HL
L00001:	ADD 	#1,@HL
!	MOV	#0,@C	
	MOV	#0,@C	
!	ADD	#2,@C
	ADD	#2,@C
!	ADD	#\&XX, @C
	ADD	#64, @C
!	.END
